app/controllers/teams/users_controller.rb

```diff
@@ -1,36 +1,64 @@
 class Teams::UsersController < AuthenticatedController
-   before_action -> { authorize team, policy_class: TeamUsersPolicy }
-
-    def remove_confirm
-      if user_present_not_current_user(user)
-        render :remove_confirm      
-      else
-        render_401
-      end
-    end
-  
-    def destroy
-      if user_present_not_current_user(user)
-        team.users.delete(user)
-        flash[:success] = I18n.t('teams.users.remove.success', email: user.email)
-        redirect_to team_path(team.id)
-      else
-        render_401
-      end
-    end
-  
-    private
+  before_action -> { authorize team, policy_class: TeamUsersPolicy }
 
-    def user
-      @user ||= team.users.find_by(id: params[:id])
-    end
+  def new
+    @user = User.new
+  end
 
-    def user_present_not_current_user(user)
-      user.present? && user != current_user
+  def create
+    new_member.user_teams << UserTeam.create!(user_id: new_member.id, group_id: team.id)
+    flash[:success] = I18n.t('teams.users.create.success', email: member_email)
+    redirect_to new_team_user_path and return
+  rescue ActiveRecord::RecordInvalid => e
+    flash[:error] = "Email '#{member_email}': " + e.record.errors.full_messages.join(', ')
+    redirect_to new_team_user_path
```

Comment:
**suggestion (non-blocking):** after playing with this locally I personally think redirecting back to the manage users page makes more sense, even if means users will have to click an extra time to add additional users. With that said, if we add the name of the team to the new user page, maybe it's not such a big deal? I don't have super strong feelings, defer to @cdelivuk and/or your thoughts!